Habitica/res/menu/chat_message.xml

```diff
@@ -10,6 +10,11 @@
         android:title="@string/send_pm" />
     <group>
         <item
+                android:id="@+id/menu.chat.copy"
+                android:icon="@drawable/ic_action_warning"
+                app:showAsAction="always|withText"
+                android:title="@string/copy_chat_message" />
```

Comment:
Sorry! When I merged the latest changes from the repo that was a conflict and it got removed. committing it now